src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -39,6 +39,7 @@ namespace Microsoft.Terminal.Settings.Editor
 
         OBSERVABLE_PROJECTED_APPEARANCE_SETTING(String, FontFace);
         OBSERVABLE_PROJECTED_APPEARANCE_SETTING(Single, FontSize);
+        OBSERVABLE_PROJECTED_APPEARANCE_SETTING(Double, LineHeight);
```

Comment:
one's a single, one's a double!

Reply:
`Double` is technically the better one, because that's what WinUI's setters and getters use. But we use `float` and `Single` throughout the project (which is alright - we don't need more than 2-3 decimal places for anything), including for the `FontSize` property. `LineHeight` on the other hand is a custom getter/setter and I felt like it'd be better to use `Double` if we can when interfacing with WinUI.